src/PrestaShopBundle/Resources/config/services/bundle/serializer.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  _defaults:
+    public: true
+
+  # Snake case normalizer definition
+  prestashop.bundle.snake_case_normalizer.object:
+    class: Symfony\Component\Serializer\Normalizer\ObjectNormalizer
+    arguments:
+      - ~
+      - '@serializer.name_converter.camel_case_to_snake_case'
+
+  # Snake case serializer definition
```

Comment:
Nice!

Reply:
Not sure if its needed though 😄 more here https://github.com/PrestaShop/PrestaShop/issues/15977#issuecomment-545364613

Reply:
Well I had a similar problem in one of my current PR actually.. We need to define a convention do we return json objects as snake case or camel case?

Reply:
I vote for camelCase :1st_place_medal: